.github/workflows/periodic.yaml

```diff
@@ -1,73 +0,0 @@
-# Copyright 2023 The Nuclio Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-name: Periodic
-
-on:
-  schedule:
-  - cron:  '0 */12 * * *'
-
-env:
-  DOCKER_BUILDKIT: 1
-
-jobs:
-  periodic:
-    name: Periodic Regression
-    runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        command: [nuctl-tests,dotnet-tests,golang-tests,java-tests,python-tests,python-runtime-tests,fast-tests]
-
-    # let's not run this on every fork, comment this out when developing periodic on your fork
-    if: github.repository == 'nuclio/nuclio'
-
-    steps:
-    - uses: actions/checkout@v4
-
-    - name: Freeing up disk space
-      run: "${GITHUB_WORKSPACE}/hack/scripts/ci/free-space.sh"
-
-    - uses: actions/setup-go@v5
-      with:
-        cache: true
-        go-version-file: go.mod
-
-    - name: Build
-      run: make build
-      env:
-        NUCLIO_NUCTL_CREATE_SYMLINK: false
-
-    - name: Test
-      run: LIST_TESTS_MAKE_COMMAND=${{ matrix.command }} make test
-      env:
-        NUCLIO_CI_SKIP_STRESS_TEST: true
-
-    - name: System stats on failure
-      if: ${{ failure() }}
-      run: "${GITHUB_WORKSPACE}/hack/scripts/ci/system-stats.sh"
```

Comment:
I added this when we had some issues with the Github runners, we might still want to run this on failure.

Reply:
and we still do here https://github.com/nuclio/nuclio/blob/a9f8aac9d324cae438273b59c15bbad67b800499/.github/workflows/long_ci.yaml#L90-L92

Reply:
right. missed that